front/src/config/i18n/en.json

```diff
@@ -1083,7 +1083,7 @@
         "discoveringError": "An error occurred, check scanner configuration, or check logs."
       },
       "setup": {
-        "title": "LANManager settings",
```

Comment:
LANManager is the name of the integration, are we sure we don't keep the name of the integration here?

Reply:
Yes, compared with the other tabs in this integration and with what's translated into French, I found it more coherent. 
Or we could rename the tabs LAN Manager Discover, LAN Manager Devices, LAN Manager settings... (and the same in French language)
As you prefer

Reply:
Ok 👍